hedera-mirror-importer/src/main/resources/db/scripts/init.sh

```diff
@@ -46,7 +46,8 @@ psql -d "user=postgres connect_timeout=3" \
   --set "rosettaPassword=${ROSETTA_PASSWORD:-mirror_rosetta_pass}" \
   --set "rosettaUsername=${ROSETTA_USERNAME:-mirror_rosetta}" \
   --set "web3Password=${WEB3_PASSWORD:-mirror_web3_pass}" \
-  --set "web3Username=${WEB3_USERNAME:-mirror_web3}" <<__SQL__
+  --set "web3Username=${WEB3_USERNAME:-mirror_web3}" \
+  --set "tempSchema=${HEDERA_MIRROR_IMPORTER_DB_TEMPSCHEMA:-temporary}" <<__SQL__
```

Comment:
I think only the init script in charts needs the `HEDERA_MIRROR_IMPORTER_` prefix, right?
 ```suggestion
   --set "tempSchema=${DB_TEMPSCHEMA:-temporary}" <<__SQL__
```

Reply:
I will make consistent. You are right that only the chart values need to do this.